bindings/wallet-cordova/src/android/WalletPlugin.java

```diff
@@ -127,6 +130,17 @@ private void walletTotalFunds(JSONArray args, CallbackContext callbackContext) t
         }
     }
 
+    private void walletId(JSONArray args, CallbackContext callbackContext) throws JSONException {
+        Long walletPtr = args.getLong(0);
+
+        try {
+            byte[] id = Wallet.id(walletPtr);
+            // TODO
```

Comment:
`TODO` what?

Reply:
@enzoc4 I am not so sure what to add here for the return type. Ideally it should be hexadecimal encoded string I suppose since this is json and this is how it is used against the REST APIs.